docs/tutorial/file.rst

```diff
@@ -305,4 +322,4 @@ particular, high frequency channels tend to take up a lot of space, and aren't a
 
     file.save_as("no_hf.h5", omit_data={"Force HF/*"})  # Omit high frequency force data from export
 
-We use `fnmatch` patterns for specifying which fields to omit from the saved `h5` file.
+We use `fnmatch` patterns for specifying which fields to omit from the saved `h5` file; the ``*`` in the line of code above means that any channel starting with ``""Force HF/""`` should be omitted from the file.
```

Comment:
👌

Reply:
Might be nice to link the `fnmatch` to here: https://docs.python.org/3/library/fnmatch.html in case people want to look up the exact syntax.

```suggestion
We use `fnmatch <https://docs.python.org/3/library/fnmatch.html>`_ patterns for specifying which fields to omit from the saved `h5` file; the ``*`` in the line of code above means that any channel starting with ``""Force HF/""`` should be omitted from the file.
```